tests/layer_tests/pytorch_tests/test_roi_align.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+import pytest
+import torch
+
+from pytorch_layer_test_class import PytorchLayerTest
+from torchvision.ops import roi_align
+
+
+class TestROIAlign(PytorchLayerTest):
+    def _prepare_input(self):
+        return (self.input_tensor, self.boxes)
+
+    def create_model(self, output_size, spatial_scale, sampling_ratio, aligned):
+
+        class torchvision_roi_align(torch.nn.Module):
+            def __init__(self, output_size, spatial_scale, sampling_ratio, aligned):
+                super().__init__()
+                self.output_size = output_size
+                self.spatial_scale = spatial_scale
+                self.sampling_ratio = sampling_ratio
+                self.aligned = aligned
+
+            def forward(self, input_tensor, rois):
+                return roi_align(
+                    input_tensor,
+                    rois.to(dtype=input_tensor.dtype),
+                    self.output_size,
+                    self.spatial_scale,
+                    self.sampling_ratio,
+                    self.aligned,
+                )
+
+        ref_net = None
+
+        return (torchvision_roi_align(output_size, spatial_scale, sampling_ratio, aligned),
+                ref_net, "torchvision::roi_align")
+
+    @pytest.mark.parametrize('input_tensor', (np.random.randn(4, 4, 4, 4).astype(np.float32),))
```

Comment:
4,4,4,4 as input is not great, this can hide some issues, could you make this input dimensions bigger?

Reply:
done